question24/main.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const agreementLinkBtn = document.getElementById('js-agreement-link');
```

Comment:
Q
There is a button in the variable name,
I think it's actually a text link to the Terms of Service, but what about it?